wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -0,0 +1,65 @@
+# Livestreaming osu!
+
+This guide will show you how to livestream osu! on Twitch using [OBS Studio](https://obsproject.com/ "Open Broadcaster Software"). Doing so is relatively simple, although it might require a reliable computer able to both run and livestream osu!.
```

Comment:
reliable ... able

```suggestion
This guide will show you how to livestream osu! on Twitch using [OBS Studio](https://obsproject.com/ "Open Broadcaster Software"). Doing so is relatively simple, although it might require a powerful computer able to run and livestream osu! at the same time.
```